zsh/.zshrc

```diff
@@ -257,6 +272,17 @@ if [[ -z "$LS_COLORS" ]]; then
   export LS_COLORS='di=1;34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
 fi
 
+# Should this be inside load-our-ssh-keys()?
+if ! can_haz ssh-askpass; then
+  echo
+  echo "For the quickstart to prompt for your ssh key passphrase on the command line,"
+  echo "enable the following zqs setting: "
+  echo $(tput setaf 2) "zqs enable-ssh-askpass-require true" $(tput setaf 7)
+  echo "Or install a SSH askpass gui application, which will prompt you for your passhphrase,"
+  echo "which is the default behavior for ssh-add"
+  echo
+fi
```

Comment:
Should this be a function too?

Reply:
Make it a function so you can call it both from `load-our-ssh-keys()` and from `zsh-quickstart-enable-ssh-askpass-require()` when the user tries enabling with `ssh-askpass` missing from their `$PATH`.

Reply:
Yeah that makes it more reusable.

Reply:
Now done using `tput sgr0`